awxkit/awxkit/cli/utils.py

```diff
@@ -22,7 +22,7 @@
     'successful': 'green',
     'failed': 'red',
     'error': 'red',
-    'cancelled': 'grey',
+    'canceled': 'grey',
```

Comment:
I think this is specific to the actual job status, which is going to be data that already exists in our database (so this is going to break something).

Reply:
Actually, never mind, this *is* correct:

https://github.com/ansible/awx/blob/devel/awx/main/models/unified_jobs.py#L74